spring-boot-parent/pom.xml

```diff
@@ -15,6 +15,50 @@
     <artifactId>spring-boot-parent</artifactId>
     <packaging>pom</packaging>
 
+    <properties>
+        <image.path>localhost:5000/${project.artifactId}</image.path>
```

Comment:
ten image.path byl chyba ustawiany w glownym pomie to pewnie stamtad mozna usunac

Reply:
Wydaje mi się, że usunąłem.